Python/Python.sublime-syntax

```diff
@@ -84,9 +84,13 @@ contexts:
     - match: \b(exec)\b(?! *($|[,.()\]}]))
       scope: keyword.other.exec.python
     - match: \b(return)\b
-      scope: keyword.control.flow.return.python
-    - match: \b(break|continue|pass)\b
-      scope: keyword.control.flow.python
+      scope: keyword.control.flow.python keyword.control.flow.return.python
```

Comment:
It seems a little odd to duplicate the `keyword` scope to remove a "scope atom" before the "scope suffix". Do your color schemes really have to target `keyword.control.flow.python`? Using a selector like `source.python keyword.control.flow` would have the same effect and remove the need for the duplication changes here, if you really must use language specific colors.

Reply:
This would be specific to the `return` right? I tried to keep a pattern for `return`, `break`, `pass` and `continue`. For the break/pass/continue scopes I had to keep the `flow.python` scope and thus add the new scopes. For `return` this is not strictly necessary, but it seemed good to have the pattern. 

I don't mind removing it for `return`, but for the others the duplication is necessary unless we address the language-suffix topic for the whole file which I did not want to do just for this small addition. I see the suffix pattern in many other syntaxes and I would certainly prefer to not have the language suffix, since that would have made things simpler and the duplication completely unnecessary.

Reply:
Usually, you just append the suffix at the end of whatever scope you have at that point. Scope names may change, and they do so rather frequently, so I don't think we need a backwards compat for this change. Suffixes in selectors are to be used when you need to target something *very specific* and may need to be updated occasionally.